build-in-docker/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM jetty:9-jdk8-openjdk as build
```

Comment:
You can use here the official maven image instead of jetty and skip the installation of maven:
```Dockerfile
FROM maven:3-eclipse-temurin-8

RUN git clone --recursive https://github.com/eea/geonetwork-eea.git /tmp/geonetwork-eea
WORKDIR /tmp/geonetwork-eea

RUN mvn -B clean install -DskipTests
RUN mvn -B -f web/pom.xml package -DskipTests -Penv-catalogue

FROM jetty:jetty:9-jdk8-eclipse-temurin
...
```